portfolio/src/main/webapp/script.js

```diff
@@ -14,6 +14,9 @@
 
 /* eslint-disable no-unused-vars */
 
+let lastLimit = -1;
```

Comment:
Can you pull these parameters from html? It's the current limit and language chosen, right?

Reply:
I think I can. Now working for it :)